docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,22 +1,156 @@
+
+```
+{
+ 'cody.contextFilters': {
+ // All repositories starting with "github.com/sourcegraph/" are allowed
+ include: [{repoNamePattern: '^github\\.com\\/sourcegraph\\/.+'}],
+ // except for "github.com/sourcegraph/cody-analytics"
+ // and the ones containing "cody" in their name.
+ exclude: [
+ {repoNamePattern: '^github\\.com\\/sourcegraph\\/cody-analytics$'},
+ {repoNamePattern: '.*cody.*'}
+ ]
+ }
+}
+```
+
+In this case, Cody can access content from repositories whose names start with `github.com/sourcegraph/`. However, it excludes files from repositories with `cody` in their name.
+
+## Cody Context Filters Compatibility Matrix
+
+<Callout type="info"> Cody Context Filters work only on Sourcegraph versions >=5.4.0, VS Code `>=1.20.0` and JetBrains `>=6.0.0 for Enterprise users.</Callout>
+
+Depending on the client type, here's a breakdown of versions supported and the behavior for unsupported versions:
+
+|        **Client Versions**        | Sourcegraph `v<5.4.0` | Sourcegraph `v=5.4.0`                           | Sourcegraph `v=5.4.X`                           | Sourcegraph `v>=5.4.0` |
+| --------------------------------- | --------------------- | ------------------------------------------------------------------------ | ------------------------------------------------------------------------- | ---------------------- |
+| **JetBrains `<= 5.5.9`**          | N/A          | Respects policy, but the server returns an unhelpful error | Respects policy, the server returns compatibility check and prompt to upgrade | Not affected           |
+| **`5.5.10 <= JetBrains < 6.0.0`** | N/A          | Respects policy, but the server only returns a compatibility check           | Respects policy, the server returns compatibility check and prompts to upgrade | Not affected           |
+| **JetBrains `>= 6.0.0`**          | N/A          | Respects policy                                                          | Respects policy                                                           | Not affected           |
+| **VS Code `< 1.16.X`**            | N/A          | Respects policy, but the server returns an unhelpful error | Respects policy, the server returns compatibility check and prompt to upgrade | Not affected           |
+| **`1.18.0 < VS Code < 1.20.0`**   | N/A          | Respects policy, but the server only returns a compatibility check           | Respects policy, the server returns compatibility check and prompt to upgrade | Not affected           |
+| **VS Code `>= 1.20.0`**           | N/A          | Respect policy                                                           | Respects policy                                                           | Not affected           |
+
```

Comment:
@MaedahBatool - I'm not sure if this is helpful, it feels a touch too verbose and not straightforward. 

Here's my suggestion:
1. Anything that is supported let's use the a checkmark instead of 'Not affected' or 'Respects policy'
2. For unsupported versions, let's capture it as a X. Instead of the verbosity, let's go with a FAQ on what happens if unsupported versions are run.

@kalanchan Thoughts?

Reply:
Tackled :)